torchgeo/datasets/benin_cashews.py

```diff
@@ -279,17 +283,20 @@ def _validate_bands(self, bands: Tuple[str, ...]) -> None:
                 raise ValueError(f"'{band}' is an invalid band name.")
 
     @lru_cache(maxsize=128)
-    def _load_all_imagery(self, bands: Tuple[str, ...] = ALL_BANDS) -> Tensor:
+    def _load_all_imagery(self, bands: Tuple[str, ...] = ALL_BANDS
+                          ) -> Tuple[Tensor, rasterio.Affine, CRS]:
         """Load all the imagery (across time) for the dataset.
 
         Optionally allows for subsetting of the bands that are loaded.
 
         Args:
             bands: tuple of bands to load
 
-        Returns
-            imagery of shape (70, number of bands, 1186, 1122) where 70 is the number of
-                points in time, 1186 is the tile height, and 1122 is the tile width
+        Returns:
+            imagery of shape (70, number of bands, 1186, 1122) where 70 is the number 
+                of points in time, 1186 is the tile height, and 1122 is the tile width
+            rasterio affine transform, mapping pixel coordinates to geo coordinates
+            coordinate reference system of tile_transform 
```

Comment:
In Sphinx, "Args" and "Returns" have very different format. "Args" can have multiple strings, and long lines should be indented to denote that it is a continuation. "Returns" can only have a single string and on indentation should be used (so this was actually broken before, not that it makes it to the generated docs). So I would remove indentation and make this a single paragraph/sentence.